website/docs/r/rights_bundle.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: "vcfa"
+page_title: "VMware Cloud Foundation Automation: vcfa_rights_bundle"
+sidebar_current: "docs-vcfa-resource-rights-bundle"
+description: |-
+ Provides a VMware Cloud Foundation Automation Rights Bundle resource. This can be used to create, modify, and delete Rights Bundles.
+---
+
+# vcfa\_rights\_bundle
+
+Provides a VMware Cloud Foundation Automation Rights Bundle resource. This can be used to create, modify, and delete Rights Bundles.
+
+## Example Usage
+
+```hcl
+resource "vcfa_rights_bundle" "new-rights-bundle" {
+  name        = "new-rights-bundle"
+  description = "new rights bundle from CLI"
+  rights = [
+    "Content Library: View",
+    "Content Library Item: View",
+    "Group / User: View",
+    "IP Blocks: View",
+  ]
+  publish_to_all_tenants = false
```

Comment:
Can't make this sample work:

![Screenshot 2025-01-28 at 11 23 45](https://github.com/user-attachments/assets/a8ca8729-6dfa-4c69-ba14-db7a3f90987e)

Reply:
Well spotted 🤦 
This is due to the rename mentioned above, now the example matches the arguments described in that same document.